### ISSUE TITLE
ERM-S1 and ERM-S2 are colocated

### ISSUE THREAD

**Issue by pmpowers-usgs** [usgs/nshmp-haz-fortran#14]
Reported by @treyapel and @nileshshome

@olboyd-usgs or @mpetersen-usgs can you confirm this and provide an updated a-grid file for ERM-S2?

The a-grids below share the same points. Although there is no specific language in the [OFR](http://pubs.usgs.gov/of/2014/1091/) discussing the two separate southern variants of the Eastern Rift Margin, it seems that S1 and S2 should be different per the comparison with CEUS-SSCn documentation as shown in the attached figure.

```
conf\CEUS\gridded\GR_DOS\ERM-S1_RLME.xls.agrid
conf\CEUS\gridded\GR_DOS\ERM-S2_RLME.xls.agrid
```

![erm-s1-s2](https://cloud.githubusercontent.com/assets/2379604/11049555/58fecf28-86fc-11e5-99e8-28c52824a871.jpg)

**Comment by olboyd-usgs** [comment]
Yes, somehow the polygonal coordinates of SRP (ERM-S2) were overwritten with SCC (ERM-S1) during an early stage of production leading to a collocation of the sources. Updated files will be uploaded.

**Comment by pmpowers-usgs** [comment]
closed in #15
